src/pages/MyToken/MyToken.jsx

```diff
@@ -1,7 +1,168 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import My from '../../components/My/My.jsx';
+import tokenImg from '../../images/kulture-token.png';
+import { M } from '../../components/My/My';
+import { T } from '../../components/Token';
+import { S } from './MyToken';
 
 const MyToken = () => {
-  return <div>MyToken</div>;
+  const [chargeAmount, setChargeAmount] = useState(0);
+  const [paymentKRW, setPaymentKRW] = useState(0);
+  const [HistoryList, setHistoryList] = useState([]);
```

Comment:
```suggestion
  const [historyList, setHistoryList] = useState([]);
```
컴포넌트의 이름만 대문자로 처리해주세요!